src/commands/new.ts

```diff
@@ -68,18 +99,25 @@ export default class New extends Command {
 
     if (questions.length) {
       const answers: any = await inquirer.prompt(questions);
+
       if (!fileName) {fileName = answers.filename as string;}
+      if (!selectedTemplate) {selectedTemplate = answers.selectedTemplate as string;}
       if (openStudio === undefined) {openStudio = answers.studio;}
     } else {
       fileName = DEFAULT_ASYNCAPI_FILE_NAME;
+      selectedTemplate = DEFAULT_ASYNCAPI_TEMPLATE;
     }
```

Comment:
I think that here should be made fallback to the default values, outside else:

```suggestion
    }
    fileName = fileName || DEFAULT_ASYNCAPI_FILE_NAME;
    selectedTemplate = selectedTemplate || DEFAULT_ASYNCAPI_TEMPLATE;
```

Reply:
Cool thanks @magicmatatjahu , made the change 👍